CHANGELOG.md

```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.13.0
+
+_Jul 7, 2022_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` are moving to beta
+
+- ✨ Allow to infer detail panel height from content (#5163) @m4theushw
```

Comment:
@m4theushw if you want to improve the example

Reply:
Isn't it more readable if we just say - Allow dynamic height for detail panel? Also, I think Matheus is on PTO until Friday.